examples/hosts/app-integration/external-data/README.md

```diff
@@ -6,15 +6,68 @@ Please note that the ideas explored here are experimental and under development.
 
 ## Strategy overview
 
-TBD
+In order to accomplish the goals above, we have split up the responsibilities into a few different pieces:
+
+**External Data Service**
+
+Many services that would hold the "source of truth" data offer explicit commit style interfaces (e.g. vi REST call or similar) which are not well suited to rapid updates.
+However, they often expose third-party integration via REST APIS for querying and manipulating data, as well as webhooks for watching updates to the data.
+
+This repo contains a service that mocks the external "source of truth" data server. This mock service offers a REST API collection and webhook interfaces in ./src/mock-external-data-service. The API requests served by this "external" service are the following:
+
+1. POST `/register-for-webhook`: Register's the sender's URL (in this case the Customer Service's URL) to receive notifications when the external task-list data changes.
+2. GET `/fetch-tasks`: Fetches the task list from the external data store.
+3. POST `/set-tasks`: Updates external data store with new tasks list (complete override).
+4. POST `/debug-reset-task-list`: Resets the external data to its original contents.
+
+Find the details of the API in the [External Data Service README](./src/mock-external-data-service/README.md)
+
+**Customer Service**
+
+Next we need a customer service that functions as the intermediary between the External Data Service and the Fluid Service. This server is responsible for authenticating to the external service on the customer's behalf. It registers to the External Data Service webhooks and listens for incoming changes. It also acts as a translation layer, tasnlating to and from the External Data Service expected format and the Fluid Service's expected format.
+
+In this example, the Customer Service contains the following endpoints:
+
+1. POST `/register-for-webhook`: Register's the sender's URL (in this case, the Fluid Service's URL) to receive notifications when the external task-list data changes.
+2. POST `/echo-external-data-webhook`: "Echoes" the external data services data update notifications to our own webhook subscribers. This data will be forwarded to our own subscribers.
+
+Find the details of the API in the [Customer Service README](./src/mock-customer-service/README.md).
+
+Next we come to how the Fluid collaboration session and clients will consume the APIs above and render the data to the screen. This example repo explores two routes to accomplish this. One is the Echo Webhook Pattern and the other is the Bot Pattern. Both are documented in more detail below.
 
 ### Echo Webhook Pattern
 
-TODO: Document the echo-webhook pattern and how it is surfaced in the example code.
+<img width="1356" alt="image" src="https://user-images.githubusercontent.com/6777404/216415477-14d0b193-29c9-48e5-8b6b-0a549a5dde58.png">
```

Comment:
Nit: For the `alt` property of these images, we should probably differentiate between the different diagrams rather than labeling them all as "image".

Reply:
Good point, I'll update them!